perfscale_regression_ci/scripts/network/network_policy_scalability.sh

```diff
@@ -17,66 +17,70 @@ source ../common.sh
 source network_policy_scalability_env.sh
 
 pass_fail=0
-workload_template_list=("${DIR}/../../kubeburner-object-templates/scaling-network-policy-config.yaml" "${DIR}/../../kubeburner-object-templates/scaling-no-network-policy-config.yaml")
+workload_template="${DIR}/../../kubeburner-object-templates/scaling-network-policy-config.yaml"
 pod_scale_max=2000
-kube_burner_job_list=("np-issue-test" "no-np-issue-test")
+kube_burner_job_list="np-issue-test"
 final_time_list=()
 deny_network_policy="${DIR}/../../content/deny_network_traffic_policy.yaml"
 deny_traffic_code=000
 return_traffic_code=200
 
+scale(){
+    echo "[INFO] Scale to $1 replicas."
+    oc scale --replicas $1 deployment network-policy-scalability -n $namespace
+}
+
 
 echo "======Use kube-burner to load the cluster with test objects======"
 
+echo "Starting scale-up test for $workload_template..."
+export NAME=${kube_burner_job_list}
+export NAMESPACE=${kube_burner_job_list}
+deployment_name=${kube_burner_job_list}
+export WORKLOAD_TEMPLATE=$workload_template
+run_workload
+
 for ((i = 0; i < 2; i++));
```

Comment:
It seems the for loop is not needed here anymore if kube_burner_job_list has only one item.